src/ui/Facet/Facet.ts

```diff
@@ -1831,29 +1847,38 @@ export class Facet extends Component {
   }
 
   private buildMore(): HTMLElement {
-    let more: HTMLElement;
     const svgContainer = $$('span', { className: 'coveo-facet-more-icon' }, SVGIcons.icons.arrowDown).el;
     SVGDom.addClassToSVGInContainer(svgContainer, 'coveo-facet-more-icon-svg');
-    more = $$('div', { className: 'coveo-facet-more', tabindex: 0 }, svgContainer).el;
+    const more: HTMLElement = $$('div', { className: 'coveo-facet-more', tabindex: 0 }, svgContainer).el;
 
     const moreAction = () => this.handleClickMore();
     $$(more).on('click', moreAction);
```

Comment:
Couldn't you use the `AccessibleButton` action functions here e.g. `withClickAction` ?

Reply:
Good idea, I will change `makeAccessibleButton` to accept an object that has 4 properties.